holoviews/element/chart.py

```diff
@@ -309,12 +309,6 @@ class Points(Chart):
 
     _min_dims = 2                      # Minimum number of columns
 
-    def __iter__(self):
-        i = 0
-        while i < len(self):
-            yield tuple(self.data[i, ...])
-            i += 1
-
 
```

Comment:
This was old and broken because it assumed the data was always an array. There is an existing issue about adding an iterator method to all the ``Dataset`` interfaces somewhere.